pom.xml

```diff
@@ -40,10 +40,10 @@
     </modules>
 
     <properties>
-        <gravitee-bom.version>2.5</gravitee-bom.version>
-        <gravitee-node.version>1.24.0</gravitee-node.version>
-        <gravitee-plugin.version>1.23.1</gravitee-plugin.version>
-        <gravitee-alert-api.version>1.9.0</gravitee-alert-api.version>
+        <gravitee-bom.version>2.8</gravitee-bom.version>
+        <gravitee-node.version>2.0.0-alpha.3</gravitee-node.version>
```

Comment:
I guess that we need to wait the next stable release to merge this one ?

Reply:
@passionne I changed it according to @phiz71 last indication.
We'll be able to release an alpha